packages/wdio-cli/src/setup.js

```diff
@@ -21,6 +21,9 @@ export default function setup () {
         }
         if (answers.installServices) {
             packagesToInstall = packagesToInstall.concat(answers.services)
+            if (answers.services.includes('wdio-chromedriver-service')) {
```

Comment:
I think this should be moved to wdio chromedriver service itself

Reply:
This is the correct way to do this, but I why can't `chromedriver` be a dependency of `wdio-chromedriver-service` that way the user doesn't have to manually add it.